WalletWasabi.Gui/Controls/WalletExplorer/SendTabViewModel.cs

```diff
@@ -536,7 +536,7 @@ private void SetFeesAndTexts()
 					// In decimal ',' means order of magnitude.
 					// User could think it is decimal point but 3,5 means 35 Satoshi.
 					// For this reason we treat ',' as an invalid character.
-					if (!UserFeeText.Contains(",") && decimal.TryParse(UserFeeText, out decimal userFee))
+					if (!UserFeeText.Contains(",") && decimal.TryParse(UserFeeText, out decimal userFee) && userFee > 0 && (userFee * 1_000) < Constants.MaxSatoshiesSupply)
```

Comment:
I would create a helper method in the ViewModel and use it here and in ValidateUserFeeText.